### ISSUE TITLE
The column chart starts with the adjusted value

### ISSUE THREAD

**Issue by cuongceg** [ehsannarmani/ComposeCharts#29]
Can my column in the Column Chart start with a value different from 0?

**Comment by ehsannarmani** [comment]
@cuongceg Hi bro, i did not understand what you mean.

**Comment by cuongceg** [comment]
@ehsannarmani Sorry for my bad English bro. So in my case, I want my column to start with a different value like in the image below 

![column_chart](https://github.com/user-attachments/assets/932b0c70-5b7b-4211-99e1-edad84cfa8f6)

Can your Column Chart support this?

**Comment by ehsannarmani** [comment]
@cuongceg yes you can specify min value for column charts, but it can constant, not dynamic for every bar.
In your example it seems min value is dynamic and its not constant.

**Comment by cuongceg** [comment]
@ehsannarmani thanks a lot for helping me. I had solution for my case. I will close this issue soon.
